### ISSUE TITLE
Size of the Comments field/scrolling within the window

### ISSUE THREAD

**Issue by Bdavy92960** [OpenAero/main#266]
If the comments field under Sequence Info exceeds 9 lines, the text begins to overwrite the Filename/Alerts box on screen, rather than creating a scroll bar to scroll down to additional text.

I recognize this may not be an issue for short comments to be printed at the top of the screen, but we want to be able to use it to track comments on draft sequences being shared before publication.  Once finalized, the notes will be deleted, not printed at the top of the page anyway.  Does not appear to be a limitation on the amount of text we can enter, just the presentation of the text on screen.

**Comment by OpenAero** [comment]
Confirmed. Will create a fix for the next update.

**Comment by OpenAero** [comment]
An update is now published on GitHub and devel. This adds a scrollbar to the Notes box when the notes exceed 4 lines.

**Comment by Bdavy92960** [comment]
Outstanding!  Looks great.

I deleted my fork to delete 5 commits that were apparently the problem.  Couldn’t figure out how to just delete the commits because they kept coming back everytime I pushed my desktop to my fork.  I created a commit with the new rules23-imac.js file.  Everything seems to be working now as my writers expect, so once the new rules file goes to the devel server, I’ll let them know and we’ll give it a couple days run time.  I think this time we may have it.

Brad
